gn2/wqflask/templates/base.html

```diff
@@ -33,262 +46,297 @@
-                        <ul>
                             <li>
-                                <a href="http://citg.uthsc.edu">
-                                    The UT Center for Integrative and Translational Genomics
+                                <a href="/collections/list">Collections
+                                    <span class="badge badge-info">{{ num_collections() }}</span>
                                 </a>
                             </li>
                             <li>
-                                <a href="https://www.nigms.nih.gov/">NIGMS</a>
-                                Systems Genetics and Precision Medicine Project (R01 GM123489, 2017-2026)
-                            </li>
-                            <li>
-                                <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2118709">NSF</a>
-                                Panorama: Integrated Rack-Scale Acceleration for Computational Pangenomics
-                                (PPoSS 2118709, 2021-2016)
-                            </li>
-                            <li>
-                                <a href="https://www.drugabuse.gov/">NIDA</a>
-                                NIDA Core Center of Excellence in Transcriptomics, Systems Genetics, and the Addictome (P30 DA044223, 2017-2022)
-                            </li>
-                            <li>
-                                <a href="http://www.nia.nih.gov/">NIA</a>
-                                Translational Systems Genetics of Mitochondria, Metabolism, and Aging (R01AG043930, 2013-2018)
+                                <a href="/repositories"
+                                   class="dropdown-toggle"
+                                   data-toggle="dropdown"
+                                   aria-haspopup="true"
+                                   aria-expanded="false">Source Code <span class="caret"></span></a>
+                                <ul class="dropdown-menu">
+                                    <li>
+                                        <a href="https://github.com/genenetwork/genenetwork2">GN2 Source Code</a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/genenetwork/genenetwork">GN1 Source Code</a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/genenetwork/sysmaintenance">System Maintenance Code</a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/genenetwork/genenetwork2/blob/testing/doc/API_readme.md">REST API Documentation</a>
+                                    </li>
+                                </ul>
                             </li>
+                            {% if g.user_session.logged_in %}
```

Comment:
nitpick: `if logged_in() `? The advantage is that the function also checks if the token is valid

Reply:
Done!